joomla/joomla-cms
# Add layouts to plg_system_privacyconsent

Pull Request for Issue # n/a.

### Summary of Changes
the layouts for plg_system_privacyconsent (label and privacynote) are hardcoded into the field and cannot be overwritten by a template override
This PR adds layouts for both the label and for the privacynote (message). these layouts can be overwritten via template overrides.


### Testing Instructions
1. enable the privacyconsent system plugin
2. set a joomla article as privacy article in the plugin
on the front-end user registration page the privacyconsent box should show

Apply this PR

[test] on the front-end user regsitration page the privacyconsent box should be exactly the same as without this PR.

3. create a template override via the template (e.g. protostar):
-> tab Create Overrides
-> column Layouts
-> select plugins > system (new!)

Edit the created override file: ./templates/protostar/html/layouts/plugins/system/privacyconsent/message.php
4. change the class from alert-info to alert-warning

[Result 1] refresh on the front-end the new user registration form

Edit the created override file: ./templates/protostar/html/layouts/plugins/system/privacyconsent/label.php
5. change ```<span class="star">&#160;*</span></label>``` to ```<span class="star">&copy;*</span></label>```

[Result 2] refresh on the front-end the new user registration form

### New result
![image](https://user-images.githubusercontent.com/2733197/51695148-a2829100-2002-11e9-86ea-72e267cbc5c3.png)


### Actual result
Make sure that after applying the PR everything works as before.

### Documentation Changes Required
